.github/workflows/ci-testing-deploy.yml

```diff
@@ -1935,7 +1935,9 @@ jobs:
           name: docker-buildx-images-${{ runner.os }}-${{ github.sha }}
           path: /${{ runner.temp }}/build
       - name: load docker images
-        run: make load-images local-src=/${{ runner.temp }}/build
+        run: |
+          make load-images local-src=/${{ runner.temp }}/build
+          make info-images
```

Comment:
load-images already calls ```docker images``` which is more compact. so in this case remove the one you dislike